ansible/inventory/production/group_vars/all/vars.yml

```diff
@@ -6,3 +6,16 @@ website_host: privacylx.org
 
 # Lets Encrypt email
 le_email: admin@privacylx.org
+
+# Login, SSH public key, comment and state of users with admins rights
+# State absent removes the user account
+adm_acct:
+  andz:
+    login: andz
+    comment: anadahz
+    state: present
+    keys:
+      - "ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIOVYO2Xu1pKrax98FpZ3dauRDMqE8Z4IPSYihD5XGRaO"
+
+# Users allowed to SSH with admin rights
+adm_logins: [ andz ]
```

Comment:
I would suggest removing this variable and extracting the username from the dictionary adm_acct to remove redundancy. But it's also ok like this.